vcpkg-init/mint-standalone-bundle.ps1

```diff
@@ -1,13 +1,22 @@
-[CmdletBinding()]
+[CmdletBinding(PositionalBinding=$False)]
 Param(
-    [Parameter(Mandatory = $true)]
+    [Parameter()]
     [string]$DestinationTarballName,
-    [Parameter(ValueFromRemainingArguments = $true)]
-    [string[]]$AdditionalFiles,
     [Parameter()]
-    [string]$TempPath
+    [string]$DestinationDir,
+    [Parameter(Mandatory)]
+    [string]$TempPath,
+    [Parameter()]
+    [switch]$readonly,
+    [Parameter(ValueFromRemainingArguments = $true)]
+    [string[]]$AdditionalFiles
 )
 
+if (-not ($DestinationTarballName -or $DestinationDir)) {
+    Write-Error "Either DestinationTarballName or DestinationDir must be set."
+    throw
+}

```

Comment:
```suggestion
```
imo, this should be done with `[Parameter(Mandatory)]`

Reply:
How would that work? At least one must be supplied.

Reply:
`ParameterSetName` does it, as @strega-nil-ms suggests below.

Reply:
Hmmmm seems to work yes